src/settings.py

```diff
@@ -67,4 +67,11 @@ class TokenSettings:
     "STRK-USDC",
     "STRK-USDT",
     "STRK-DAI",
-]
\ No newline at end of file
+]
+
+COLLATERAL_TO_DEBT_TOKENS = {}
+for pair in PAIRS:
```

Comment:
Feel free to replaice PAIRS, we don't have to use it at all after the change.

Reply:
This variable is still being used in multiple files